modules/installing-ove-console-final.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-ove.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-installing-ove-console-final_{context}"]
+= Completing cluster configuration and initiating the installation
+
+Before you can finally initiate the cluster installation, you must verify host details, configure cluster networking details, and download the default cluster credentials.
+
+.Prerequisites
+
+* You have booted all of the hosts that will comprise your cluster and configured them with the correct rendezvous node IP address.
++
+[IMPORTANT]
+====
+You can add as many hosts as you want to your cluster.
+However, at this stage, you must at least have enough available hosts to match the value you specified in the *Number of control plane nodes* field of the installation console's *Cluster details* page.
+====
+
+.Procedure
+
+. On the installation console hosted by the rendezvous node, configure the hosts in the *Host discovery* page:
+
+.. Verify that every machine you booted from the ISO image appears in the *Host Inventory* section and has a *Status* value of *Ready*.
+
+.. For each host, click the expand icon and verify that all of the specification fields are correct.
+
+.. Optional: For each host except the rendezvous node, configure the role by selecting an option from the dropdown menu of the *Role* column.
+The default value for every host except the rendezvous node is `Auto-assign`.
+
+.. Click *Next* to continue.
+
+. On the *Storage* page, click the expand icon for each host and verify that all of the specification fields are correct.
+
+. Click *Next* to continue.
+
+. If you will manage your own networking, select the *User-Managed Networking* option on the *Networking* page.
```

Comment:
Out of scope for merge review, but there are lots of future tense will's in here that could likely be adjusted